pkgs/development/coq-modules/math-classes/default.nix

```diff
@@ -1,21 +1,22 @@
-{ stdenv, fetchurl, coq }:
+{ stdenv, fetchFromGitHub, coq, coqPackages }:
 
-stdenv.mkDerivation {
-  name = "coq${coq.coq-version}-math-classes-1.0.6";
+if ! stdenv.lib.versionAtLeast coq.coq-version "8.6" then
+  throw "Math-Classes requires Coq 8.6 or later."
```

Comment:
Is this true? How difficult is it to keep the old package for 8.5 alive?

Reply:
@vbgl There is no package for 8.5 in the sense that the previous package available in nixpkgs was already requiring Coq 8.6 (at least that's what the corresponding OPAM package says, I haven't checked that there was indeed a build failure).

Reply:
My bad. You’re right: `coqPackages_8_5.math-classes` currently fails to build.